messenger.go

```diff
@@ -0,0 +1,22 @@
+package plugin
+
+// MessageSendRequest describes a message to be send by MessageHandler#SendMessage
+type MessageSendRequest struct {
+	Message  string
+	Title    string
+	Token    string
```

Comment:
I would say the plugin shouldn't care about tokens, we can create an internal application for plugins in gotify/server.

Reply:
Sounds fine to me